tests/foreman/ui/test_http_proxy.py

```diff
@@ -65,8 +63,8 @@ def test_positive_create_update_delete(module_org, module_location, target_sat):
         assert http_proxy_values['http_proxy']['name'] == http_proxy_name
         assert http_proxy_values['http_proxy']['url'] == http_proxy_url
         assert http_proxy_values['http_proxy']['username'] == username
```

Comment:
Non-blocking comment: You can rephrase the above assertions in similar manner as you've done below.

Reply:
These I would rather keep since here I expect the exact values.

The reason for the changes bellow was that during proxy creation the current taxonomy is used (which I find quite useful for user), and since we stay in Default org/loc during test run, they are added too. So I need to assert that the ones added [here](https://github.com/SatelliteQE/robottelo/pull/14646/files#diff-def74d5874ed2fd5248fab2f08b6205fa66a9ef4987eea24ca1780522fb84175R57-R58) are added too.